sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsDelete.java

```diff
@@ -1,18 +1,18 @@
-/*
+/**
  * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
+ * contributors license agreements. See the NOTICE file distributed with
```

Comment:
Likewise 'contributors' is incorrect here and in any event, this is the standard header - don't change it